docs/01-app/02-guides/json-ld.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: How to use JSON-LD
+nav_title: JSON-LD
+description: Learn how to add JSON-LD to your Next.js application to describe your content.
+---
+
+[JSON-LD](https://json-ld.org/) is a format for structured data that can be used by search engines to understand your content. For example, you can use it to describe a person, an event, an organization, a movie, a book, a recipe, and many other types of entities.
```

Comment:
Used by search engines, and also AI! JSON-LD is great for AI crawlers to help them understand the structure of the page beyond pure content